webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7361,20 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.checkBotDetail = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/application/" + MCK_APP_ID + "/bot/" + CURRENT_GROUP_DATA.conversationAssignee,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                                CURRENT_GROUP_DATA.CHAR_CHECK = data.data[0] ? ((data.data[0].aiPlatform == 'dialogflow' && !((data.data[0]).autoHumanHandoff)) ? true : false) : false ;
+                            },
+                    error: function () {
+                            console.log('Unable to load bot info. Please reload page.');
```

Comment:
@hardikparashar There is no need to add the logs using console.log as we're removing all the logs while minifying these files for production builds. 
In case you want to print this log you can throw an error.